### PR TITLE
Deployment section docs structure for KF 1.4 docs

### PR DESCRIPTION
**Pull request by judyheflin**
PR for Deployment section of Kubeflow on AWS website.

Pages included: 

Deployment
    Install Kubeflow
        Prerequisites
        Cognito 
        RDS and S3
        Cognito, RDS, and S3
        Vanilla
        Add-Ons
            EFS
            FSx
            CloudWatch
            Load balancing
    Uninstall Kubeflow

### CODE REVIEW

File: docs/deployment/uninstall-kubeflow.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+```bash
+kubectl get profile
+kubectl delete profile --all
+```
+
+Then, delete the Kubeflow deployment with the following command:
+
+```bash
+kustomize build example | kubectl delete -f
```
Comment (surajkota) [inline docs/deployment/uninstall-kubeflow.md:19]: You can delete a Kubeflow deployment by running the `kubectl delete` command on the manifest according to the deployment option you chose. For example, to delete a vanilla installation, run the following command:

`kustomize build docs/deployment/vanilla/ | kubectl delete -f -`

This command assumes you have the repository in the same state as when you installed Kubeflow

File: docs/deployment/vanilla/guide.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
++++
+title = "Vanilla Installation"
+description = "Deploy Kubeflow on AWS using Amazon Elastic Kubernetes Service (EKS)"
+weight = 60
++++
+
 # Deploying Kubeflow on EKS
 
-This guide describes how to deploy vanilla Kubeflow on AWS EKS. This vanilla version has very minimal changes to the upstream Kubeflow manifests. Here are the changes
```
Comment (surajkota) [inline docs/deployment/vanilla/guide.md:3]: its being removed from here, where is it moved to?

File: docs/deployment/README.md [hunk 3]
```diff
@@ -1,92 +0,0 @@
-# Kubeflow on AWS
-
-## Deployment Options
```
Comment (surajkota) [inline docs/deployment/README.md:3]: Should this information go to index.md under deployment folder?

**Review by surajkota — DISMISSED** [review]
reivewed everything except EFS, FSx. I am not sure if these are coming from Quip doc or because of pending rebase. @mbaijal can you take a look?

File: docs/deployment/add-ons/_index.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
++++
+title = "Add-ons"
+description = "Add-on installation guides for Kubeflow on AWS"
```
Comment (surajkota) [inline docs/deployment/add-ons/_index.md:3]: There should be a place to call out add-ons are components which can be installed on top of a Kubeflow deployment

**Review by ryansteakley — COMMENTED** [review]
Changes look fine to me, waiting to see what revisions are made in response to Suraj's comments

File: docs/deployment/rds-s3/guide.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
-# Kustomize Manifests for RDS and S3
-
-## Overview
++++
+title = "RDS and S3"
+description = "Kustomize Manifests for RDS and S3"
```
[the remark this answers is not in the document]
Comment (rrrkharse) [reply docs/deployment/rds-s3/guide.md:3]: makes sense

File: docs/deployment/add-ons/_index.md (hunk 4 above)
Comment (mbaijal) [reply docs/deployment/add-ons/_index.md:3]: we can iterate on this, ok for now.

File: docs/deployment/rds-s3/guide.md [hunk 6]
```diff
@@ -374,8 +376,6 @@ cd tests/e2e
 ```
 2. Install the script dependencies `pip install -r requirements.txt`
 3. Make sure you have the configuration file created by the script in `tests/e2e/utils/rds-s3/metadata.yaml`
-4. Export values for `CLUSTER_REGION` then run the script
 ```
-EXPORT CLUSTER_REGION=
-PYTHONPATH=.. python utils/rds-s3/auto-rds-s3-cleanup.py --region $CLUSTER_REGION 
+PYTHONPATH=.. python utils/rds-s3/auto-rds-s3-cleanup.py
```
[the remark this answers is not in the document]
Comment (mbaijal) [reply docs/deployment/rds-s3/guide.md:380]: @rrrkharse Can you comment on this

[the remark this answers is not in the document]
Comment (rrrkharse) [reply docs/deployment/rds-s3/guide.md:380]: i don't think judy's PR has the latests main branch changes, so it is diffing against older changes

File: docs/deployment/uninstall-kubeflow.md (hunk 1 above)
Comment (mbaijal) [reply docs/deployment/uninstall-kubeflow.md:19]: @judyheflin you can replace lines 16 to 20 with the content Suraj posted here

File: docs/deployment/add-ons/storage/fsx-for-lustre/guide.md [hunk 7]
```diff
@@ -1,37 +1,77 @@
-# Using Amazon FSx as Persistent Storage with Kubeflow
++++
+title = "FSx for Lustre"
+description = "Use Amazon FSx as persistent storage with Kubeflow on AWS"
+weight = 20
++++
 
 This guide describes how to use Amazon FSx as Persistent storage on top of an existing Kubeflow deployment.
```
Comment (mbaijal) [inline docs/deployment/add-ons/storage/fsx-for-lustre/guide.md:7]: There is an intro in the quip doc for both efs and fsx but we can work on getting that in post release.

Comment (mbaijal) [reply docs/deployment/add-ons/storage/fsx-for-lustre/guide.md:7]: @judyheflin Could you track this in your backlog

**Review by rrrkharse — APPROVED** [review]

File: docs/deployment/add-ons/storage/fsx-for-lustre/guide.md (hunk 7 above)
Comment (judyheflin) [reply docs/deployment/add-ons/storage/fsx-for-lustre/guide.md:7]: Yep!

**Review by mbaijal — APPROVED** [review]

**Pull request merged**